comet/metadata/episode_index.py

```
@@ -0,0 +1,244 @@
+        )
+
+    async def _refresh_from_cinemeta(self, series_id: str) -> None:
+        try:
+            async with self.session.get(
+                _CINEMETA_SERIES_META_URL.format(series_id=series_id)
+            ) as response:
+                if response.status == 404:
+                    return
+                response.raise_for_status()
+                payload = await response.json()
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)